src/components/PhoneNumberInput/style.css

```diff
@@ -1,34 +1,79 @@
+@import (less) "../Theme/constants.css";
+
 .loading {
   color: #5D6B78;
-  font-size: 20px;
-  line-height: 32px;
-  margin: 8px;
+  font-size: 20*@unit;
+  line-height: 1.6;
+  margin: 6.4*@unit;
   float: left;
 }
 
 .mobileInput {
-  font-size: 16px;
+  font-size: 16*@unit;
 }
 
 .phoneNumberContainer {
   float: left;
   width: 100%;
-  padding: 8px 12px;
+  padding: 8*@unit 12*@unit;
 }
 
-.phoneNumberContainer :global .react-phone-number-input__icon {
-  height: 1.8em;
-  width: 1.8em;
-  border: none;
-  display: block;
-}
+.phoneNumberContainer :global {
+  .react-phone-number-input__icon {
+    height: 28.8*@unit;
+    width: 28.8*@unit;
+    border: none;
+    display: block;
+  }
 
-.phoneNumberContainer :global .rrui__select__arrow {
-  color: #2C3E4F;
-}
+  .rrui__select__arrow {
+    color: #2C3E4F;
+  }
 
-.phoneNumberContainer :global .rrui__select__options:not(.rrui__select__options--menu) {
-  width: 438px;
+  .rrui__select__options:not(.rrui__select__options--menu) {
+    width: 438*@unit;
+  }
+
+  /* We need to override the `rrui` styles from REM to EM, for our relative fonts */
+  .rrui__input:not(.rrui__input--multiline),
+  .rrui__input-label {
+    height: 1.8em;
```

Comment:
hm, shouldn't these have the @unit indirection as well?

Reply:
I'm trying to keep them as close to the library's styles that we're overriding as possible

The library was doing stuff like `calc(10px + 1em)` for reasons I don't know. We could use `@unit` I guess, I was just trying to make it remain "other"

Reply:
I think they do that to handle the round icon used for international and the square icon used for specific country flags